openedx/features/edly/tasks.py

```diff
@@ -39,7 +41,14 @@ def send_bulk_mail_to_students(students, param_dict, message_type):
         'comment_reply': CommentReplyNotification
     }
     message_class = message_types[message_type]
-    param_dict['site'] = Site.objects.get(id=param_dict['site_id'])
+    try:
+        # Here we don't know if the request is coming from LMS or CMS but we need to use the LMS Site in both cases.
+        lms_base = settings.LMS_BASE
```

Comment:
Delete this variable if you are not using this below

Reply:
Done